src/poliastro/tests/test_twobody.py

```diff
@@ -1,40 +1,32 @@
+import pytest
 import numpy as np
 from numpy.testing import assert_allclose
 
 from astropy import units as u
 
 from poliastro.bodies import Earth
-
 from poliastro.core.elements import coe2rv, coe2mee, rv2coe
 from poliastro.twobody.equinoctial import mee2coe
 
 
-def test_convert_between_coe_and_rv_is_transitive():
-    k = Earth.k.to(u.km**3 / u.s**2).value  # u.km**3 / u.s**2
+@pytest.fixture(scope="module")
```

Comment:
Didn't know about `scope="module"`, nice!

https://docs.pytest.org/en/3.8.0/fixture.html#scope-sharing-a-fixture-instance-across-tests-in-a-class-module-or-session

However, I believe that they are used for expensive fixtures, so I'd rather remove it.

Reply:
Sure! will do so!